tests/Endpoint/EnvelopeTagsEndpointTest.php

```diff
@@ -14,6 +14,12 @@ public function testCRUD(): void
         self::assertCrudRequests(self::endpoint(), '/api/envelopes/bar/tags');
     }
 
+    public function testUpdateValues(): void
+    {
+        self::endpoint()->updateValues(['foo' => 'bar']);
+        self::assertLastRequest('PUT', '/api/envelopes/bar/tags/values');
```

Comment:
Lze přidat ještě assert na tělo requestu

```suggestion
        self::assertLastRequest('PUT', '/api/envelopes/bar/tags/values',  ['foo' => 'bar']);
```